src/helpers/client/safari-client.ts

```diff
@@ -126,21 +136,39 @@ export class SafariClient implements Client {
     document.dispatchEvent(new CustomEvent(`AsperaDesktop.${type}`, payload));
   }
 
+  /**
+   * Dispatches a custom event to the document to communicate with the Safari extension.
+   * @param type The type of Safari extension event to dispatch.
+   * @param method The method name to invoke on the extension.
+   * @param payload Optional parameters for the method.
+   */
+  private dispatchPromiseEvent(type: SafariExtensionEventType, method: string, payload?: any): Promise<any> {
```

Comment:
If we don’t have type and don’t care on our side we should probably use unknown instead of any wherever possible.

Reply:
Done!